v22.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -138,6 +138,115 @@ You [can associate an EC2 instance with an IAM role](https://docs.aws.amazon.com
 aws ec2 associate-iam-instance-profile --iam-instance-profile Name={example profile} --region={us-east-2} --instance-id {instance example}
 ~~~
 
+### Assume role authentication
+
+{{site.data.alerts.callout_info}}
+CockroachDB supports assume role authentication on clusters running v22.2. Authenticating to cloud storage with `ASSUME_ROLE` on clusters running versions v22.1 and earlier, or mixed versions, is not supported and will result in failed bulk operations.
```

Comment:
@benbardin @rhu713 @livlobo I know there is still discussion going on around this, but here is the proposed note. 

Depending on the outcome of discussions, we could also add a known limitation to the docs for this (bulk ops will fail if cluster is running an earlier version than v22.2)... and then remove the limitation if/when a blocker for the earlier versions is implemented.

Reply:
This part is LGTM! I'm also in favor of keeping this section in even when we do have the version gates in place as it tells the user what's happening when they use assume role in a mixed version cluster and get an error.